package.json

```diff
@@ -20,19 +20,16 @@
     "slick-carousel": "^1.8.1"
   },
   "devDependencies": {
-    "@webdevstudios/js-coding-standards": "^1.0.0-beta1",
-    "@wordpress/eslint-plugin": "^4.0.0",
+    "@webdevstudios/js-coding-standards": "~1.0.2",
     "autoprefixer": "^9.7.4",
     "browser-sync": "^2.26.7",
     "cssnano": "^4.1.10",
-    "eslint": "^6.8.0",
```

Comment:
You don't need to install eslint-ish stuff here since js-coding-standards takes care of that for you. Could eventually cause a conflict, let js-coding-standards to the hard work.